ibr/demo.js

```diff
@@ -17,13 +17,39 @@ function onChooseFile() {
     const fr = new FileReader();
     fr.onload = function(evt) {
       const bin = evt.target.result;
-      IBRSDK.render( bin,
-          0, document.getElementById('mainCanvas') ); //top level, index 0
+      const out = IBRSDK.render( bin,
```

Comment:
we need to decouple these, the render function shouldn't be handling data.

Reply:
Decoupled saveToIBR from rendering